<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGBA::MemoryView</class>
 <widget class="QWidget" name="QGBA::MemoryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="regions"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Inspect Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="segments">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="segmentColon">
       <property name="text">
        <string notr="true">:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="setAddress">
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="prefix">
        <string notr="true">0x</string>
       </property>
       <property name="maximum">
        <number>268435455</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Set Alignment:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="width8">
       <property name="text">
        <string>&amp;1 Byte</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="width16">
       <property name="text">
        <string>&amp;2 Bytes</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="width32">
       <property name="text">
        <string>&amp;4 Bytes</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGBA::MemoryModel" name="hexfield" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QFrame" name="frame0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QComboBox" name="encoding">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>ASCII</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UTF-8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UTF-16LE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Shift-JIS</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>String:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stringVal">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadTBL">
          <property name="text">
           <string>Load TBL</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QFrame" name="frame1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>int8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="int8">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>4</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>uint8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uint8">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>3</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>int16</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="int16">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>6</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>uint16</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uint16">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>5</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>int32</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="int32">
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>uint32</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uint32">
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="copy">
       <property name="text">
        <string>Copy Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paste">
       <property name="text">
        <string>Paste</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveRange">
       <property name="text">
        <string>Save Range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGBA::MemoryModel</class>
   <extends>QWidget</extends>
   <header>MemoryModel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>regions</tabstop>
  <tabstop>segments</tabstop>
  <tabstop>setAddress</tabstop>
  <tabstop>width8</tabstop>
  <tabstop>width16</tabstop>
  <tabstop>width32</tabstop>
  <tabstop>int8</tabstop>
  <tabstop>stringVal</tabstop>
  <tabstop>loadTBL</tabstop>
  <tabstop>copy</tabstop>
  <tabstop>paste</tabstop>
  <tabstop>save</tabstop>
  <tabstop>saveRange</tabstop>
  <tabstop>load</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
